src/client/components/MealById.js

```diff
@@ -0,0 +1,158 @@
+import React, { useContext, useState } from "react";
+import Header from "./Header";
+import Footer from "./Footer";
+import { useParams } from "react-router-dom";
+import DataContext from "./DataContext";
+import { Link } from "react-router-dom";
+
+export default function MealById() {
```

Comment:
I'd be wary to name components based on the API they are using! What if the API endpoint name changes or the component gets data from something else?

Instead, name it for what it shows the user!